src/bin/collapse-dtrace.rs

```diff
@@ -7,7 +7,7 @@ use inferno::collapse::dtrace::{Folder, Options};
 use inferno::collapse::{Collapse, DEFAULT_NTHREADS};
 use once_cell::sync::Lazy;
 
-static NTHREADS: Lazy<String> = Lazy::new(|| format!("{}", *DEFAULT_NTHREADS));
+static NTHREADS: Lazy<String> = Lazy::new(|| DEFAULT_NTHREADS.to_string());
```

Comment:
I see you watched the stream :p

Want to fix this in `collapse-guess.rs` and `collapse-perf.rs` too?

Reply:
Yep, I guess I missed those.